core/controllers/base_test.py

```diff
@@ -179,15 +179,6 @@ def setUp(self):
         # Update a config property that shows in all pages.
         self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
         self.login(self.ADMIN_EMAIL, is_super_admin=True)
-        response = self.testapp.get('/admin')
-        csrf_token = self.get_csrf_token_from_response(response)
-        self.post_json('/adminhandler', {
```

Comment:
Also, note that you can run backend tests locally -- see https://github.com/oppia/oppia/wiki/Running-Tests#server-side-tests